samples/template/docker-compose.x86.yml

```diff
@@ -0,0 +1,90 @@
+version: "3.3"
+services:
+
+  client:
+    image: ghcr.io/insight-platform/savant-deepstream:latest
+    volumes:
+      - zmq_sockets:/tmp/zmq-sockets
+      - ./src/client:/opt/savant/src/client
+      - ./assets/test_data:/test_data
+      - ./src/output:/opt/savant/src/output
+    entrypoint: python /opt/savant/src/client/run.py
+    environment:
+      - MODULE_HOSTNAME=module
+    links:
+      - jaeger
+    networks:
+      - network
+
+  module:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.x86
+    volumes:
+      - zmq_sockets:/tmp/zmq-sockets
+      - ./models_cache:/models
+      - ./downloads_cache:/downloads
+      - ./src/module:/opt/savant/src/module
+    environment:
+      - HEALTHCHECK_PORT=8888
+      - ZMQ_SRC_ENDPOINT=sub+bind:ipc:///tmp/zmq-sockets/input-video.ipc
+      - ZMQ_SINK_ENDPOINT=pub+bind:ipc:///tmp/zmq-sockets/output-video.ipc
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+    networks:
+      - network
+
+  jaeger:
+    image: jaegertracing/all-in-one:1.49
+    environment:
+      - COLLECTOR_ZIPKIN_HOST_PORT=:9411
+    ports:
+      - "6831:6831/udp"
+      - "6832:6832/udp"
+      - "5778:5778"
+      - "16686:16686"
+      - "4317:4317"
+      - "4318:4318"
+      - "14250:14250"
+      - "14268:14268"
+      - "14269:14269"
+      - "9411:9411"
+    networks:
+      - network
+
+  always-on-sink:
```

Comment:
There is no mention of such a sink in the template module README.

Reply:
Added notes for Always On Sink + Uri Input